sourcegraph/sourcegraph.proto

```diff
@@ -142,7 +146,9 @@ message Repo {
 	// Fork is whether this repository is a fork.
 	bool fork = 12;
 
-	// Mirror is whether this repository is a mirror.
+	// Mirror indicates whether this repo's canonical location is on
+	// another server. Mirror repos track their upstream and are not
+	// eligible for discovery on this server.
 	bool mirror = 13;
```

Comment:
Nope. Consider an open source repo, for example. It's hosted on Sourcegraph.com. My local Sourcegraph will tell me that its origin is Sourcegraph.com but it is not a mirror because neither my local Sourcegraph nor Sourcegraph.com is mirroring it. My local Sourcegraph never actually clones it to my disk (it just makes API calls to Sourcegraph.com) and Sourcegraph.com is where it is actually hosted (not just mirrored).

Does that make sense? Let me know if this is unclear or weird. I will add that paragraph example to the docstrings. 


> On Jun 20, 2015, at 02:30, Gabriel Aszalos <notifications@github.com> wrote:
> 
> In sourcegraph/sourcegraph.proto:
> 
> > @@ -142,7 +146,9 @@ message Repo {
> >  	// Fork is whether this repository is a fork.
> >  	bool fork = 12;
> >  
> > -	// Mirror is whether this repository is a mirror.
> > +	// Mirror indicates whether this repo's canonical location is on
> > +	// another server. Mirror repos track their upstream and are not
> > +	// eligible for discovery on this server.
> >  	bool mirror = 13;
> Can it be implied that if an origin is different from your local instance, then the repo is mirrored? If so, do we still need this field?
> 
> —
> Reply to this email directly or view it on GitHub.
>